azure-pipelines-windows-template.yml

```diff
@@ -32,6 +32,11 @@ jobs:
       qmake CONFIG+="7zip" %DEFINES_VAR%
       nmake
     displayName: 'Build'
+  - script: |
+      call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\${{ parameters.vc_vars }}"
+      set PATH=C:\Qt\${{ parameters.qt_version }}\${{ parameters.qt_spec }}\bin;%PATH%
+      nmake check TESTARGS="-maxwarnings 100000"
+    displayName: 'Run tests'
```

Comment:
Unfortunately, the test output on Windows is absent from [the CI build logs](https://dev.azure.com/luisangelsm/YACReader/_build/results?buildId=441&view=logs&jobId=05ed95bb-1786-5052-b294-220ffba395f2&j=b9e62f99-1a98-5ed7-01d2-f4794231ed79&t=c8064208-5108-5c88-7a6d-4ac7b3cf2e15): 

```cmd
"C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.28.29910\bin\HostX86\x86\nmake.exe" -f Makefile.Release check

Microsoft (R) Program Maintenance Utility Version 14.28.29910.0
Copyright (C) Microsoft Corporation.  All rights reserved.

	call D:\a\1\s\tests\concurrent_queue_test\target_wrapper.bat  release\concurrent_queue_test.exe -maxwarnings 100000
Finishing: Run tests
```

I suppose either `call` or `target_wrapper.bat` suppresses `concurrent_queue_test.exe`'s output. I looked for a fix but didn't find one.